### PR TITLE
Fixes and optimizations for retry_failed.py

### LINKED ISSUE #213: retry_failed.py error: missing 'node_addr'
if you run python3 src/retry_failed.py with any arguments it gives the error:

MainThread - master configuration file not present.
Traceback (most recent call last):
  File "src/retry_failed.py", line 200, in <module>
    main(args)
  File "src/retry_failed.py", line 82, in main
    config_client_manager = SimpleClientManager(client_path)
TypeError: __init__() missing 1 required positional argument: 'node_addr'

### PR DESCRIPTION
**Pull request by utdrmac**
Fixes #213

### CODE REVIEW

**Comment by utdrmac** [comment]
@Xa52c I pushed a small fix for that. Try again please.

**Comment by Xa52c** [comment]
those 2 fixes moved me on to the next error:

No failed payment files found under directory '/home/user/tezos-reward-distributor/reports/tz1..../payments/failed'

it's using the wrong 'failed' dir. there are 2: (one under home/pymnt/reports/...) (the other under home/tezos-reward-distributor/reports/...)

this is pointing to the tezos-reward-distributor reports dir which is not used.

ok, if i move the file from pymnt/reports to tezos-reward-distributor/reports, we move to the next error.

`2020-03-27 14:26:00,940 - MainThread - Working on failed payment file /home/user/tezos-reward-distributor/reports/tz1.../payments/failed/209.csv
Traceback (most recent call last):
  File "src/retry_failed.py", line 200, in <module>
    main(args)
  File "src/retry_failed.py", line 145, in main
    p.retry_failed_payments(args.retry_injected)
  File "/home/user/tezos-reward-distributor/src/pay/payment_producer.py", line 374, in retry_failed_payments
    batch = CsvPaymentFileParser().parse(payment_failed_report_file, cycle)
  File "/home/user/tezos-reward-distributor/src/util/csv_payment_file_parser.py", line 17, in parse
    records = [self.FromPaymentCSVDictRow(row, cycle) for row in dict_rows]
  File "/home/user/tezos-reward-distributor/src/util/csv_payment_file_parser.py", line 17, in <listcomp>
    records = [self.FromPaymentCSVDictRow(row, cycle) for row in dict_rows]
  File "/home/ryan/tezos-reward-distributor/src/util/csv_payment_file_parser.py", line 22, in FromPaymentCSVDictRow
    rl = RewardLog(row["address"], row["type"], None)
TypeError: __init__() missing 1 required positional argument: 'current_balance'
`

**Comment by jdsika** [comment]
@amzid I need your approval here to merge

File: src/retry_failed.py [hunk 1]
```diff
@@ -117,7 +117,8 @@ def main(args):
         NB_CONSUMERS = 0
```
Comment (amzid) [inline src/retry_failed.py:117]: I had to edit line 91 to `    wllt_clnt_mngr = WalletClientManager(client_path, args.node_addr, contracts_by_alias, addresses_by_pkh, managers,
                                         verbose=args.verbose)`
Except for that, it looks good to me and with this little change it worked fine for me.

**Comment by jdsika** [comment]
@utdrmac we can merge after you accept the proposed change

File: src/retry_failed.py (hunk 1 above)
Comment (utdrmac) [reply src/retry_failed.py:117]: Nice catch.

**Comment by utdrmac** [comment]
All good.

**Pull request merged**